### PR TITLE
added all the new compliance images and updated the handlebars template

### PR DESCRIPTION
**Pull request by moeamaya**
@gib @sandersonet I only exported retina sized images. They're pretty small so figured that was okay, but let me know otherwise

### CODE REVIEW

**Comment by sandersonet** [comment]
I'm good with retina only--though I wonder if the compliance cycle and other custom-made graphics should be SVG?

**Comment by moeamaya** [comment]
I've never had experience with gradient and dropshadw svgs, but I presume they'd be fine right?

**Comment by sandersonet** [comment]
Yep, they should be quite small since gradients are essentially just styles on a path/shape.   Are you using Sketch? Sketch does a good job of exporting layer groups into SVG. 

You can also tidy up the number of HTTP requests by injecting the SVGs as partials, rather than referencing them with an  tag.  See the SVGs here as an example: https://support.aptible.com/quickstart/

**Comment by moeamaya** [comment]
K cool. Yea these are coming out of sketch. I like the partial idea. I'll go ahead and do the changes and let you know when they're ready, prob around my lunch time.

Thanks for the feedback Skylar!

**Comment by sandersonet** [comment]
:+1: Great work @moeamaya

File: compliance/index.hbs [hunk 1]
```diff
@@ -1,89 +1,52 @@
...
-      Our customers estimate Aptible reduces their manual compliance burden by up to 90%.
-  3:
-    title: Reduce and isolate security vulnerabilities
-    body: |-
-      Running your infrastructure means your team can focus their security efforts on application-level issues.
-  4:
-    title: Capture risk
+images:
+  risk: /pages/assets/compliance/compliance-risk.png
+  risk_testimonial: /pages/assets/compliance/jim-boss.png
```
Comment (fancyremarker) [inline compliance/index.hbs]: This absolutely needs to stay.

Comment (moeamaya) [inline compliance/index.hbs]: @fancyremarker I'm gonna merge the repo now so Sam and Chas can work on the copy.

We need to a do a little design update with this content, I'll submit a new PR in the afternoon once we have that sorted.

**Pull request merged**